File: Mobile App (Flutter)/haber_uygulamasi/lib/class/haber.dart
import 'package:http/http.dart' as baglanti;
import 'dart:async';
import 'dart:convert';

class Haber {
  int id;
  String baslik;
  String ozet;
  String icerik;
  String yayimTarihi;
  String yazarID;
  int kategoriID;
  int tipID;
  String resim;
  String kucukResim;
  int goruntulenme;
  Null aspnetUsers;
  Null kategori;
  Null tip;

  Haber(
      {this.id,
      this.baslik,
      this.ozet,
      this.icerik,
      this.yayimTarihi,
      this.yazarID,
      this.kategoriID,
      this.tipID,
      this.resim,
      this.kucukResim,
      this.goruntulenme,
      this.aspnetUsers,
      this.kategori,
      this.tip});

  Haber.fromJson(Map<String, dynamic> json) {
    id = json['Id'];
    baslik = json['Baslik'];
    ozet = json['Ozet'];
    icerik = json['Icerik'];
    yayimTarihi = json['YayimTarihi'];
    yazarID = json['YazarID'];
    kategoriID = json['KategoriID'];
    tipID = json['TipID'];
    resim = json['Resim'];
    kucukResim = json['KucukResim'];
    goruntulenme = json['Goruntulenme'];
    aspnetUsers = json['aspnet_Users'];
    kategori = json['Kategori'];
    tip = json['Tip'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['Id'] = this.id;
    data['Baslik'] = this.baslik;
    data['Ozet'] = this.ozet;
    data['Icerik'] = this.icerik;
    data['YayimTarihi'] = this.yayimTarihi;
    data['YazarID'] = this.yazarID;
    data['KategoriID'] = this.kategoriID;
    data['TipID'] = this.tipID;
    data['Resim'] = this.resim;
    data['KucukResim'] = this.kucukResim;
    data['Goruntulenme'] = this.goruntulenme;
    data['aspnet_Users'] = this.aspnetUsers;
    data['Kategori'] = this.kategori;
    data['Tip'] = this.tip;
    return data;
  }
}

Future<List<Haber>> habergetirhepsi(String yazi) async {
  final String baseUrl = "http://192.168.1.10/HaberApi/api";
  final response = await baglanti.get("$baseUrl/" + yazi);
  var list = json.decode(response.body) as List;
  List<Haber> haber = list.map((i) => Haber.fromJson(i)).toList();
  if (response.statusCode == 200) {
    return haber;
  } else {
    return null;
  }
}

Future<Haber> habergetirtek(int deger) async {
  final String baseUrl = "http://192.168.1.10/HaberApi/api";
  final response = await baglanti.get("$baseUrl/haber/" + deger.toString());
  if (response.statusCode == 200) {
    return Haber.fromJson(json.decode(response.body));
  } else {
    return null;
  }
}

File: MVC WebSites & API (ASP.NET)/HaberPortal/HaberApi/Models/Haber.cs
namespace HaberApi
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;
    using System.Web;
    using System.Web.Mvc;

    [Table("Haber")]
    public partial class Haber
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Haber()
        {
            Resim1 = new HashSet<Resim>();
        }

        public int Id { get; set; }

        [Required(ErrorMessage ="Baþlýk alanýný gereklidir.")]
        [StringLength(150)]
        public string Baslik { get; set; }

        [Required(ErrorMessage ="Özet alaný gereklidir.")]
        [StringLength(300)]
        public string Ozet { get; set; }
        [AllowHtml]
        [Column(TypeName = "ntext")]
        [Required(ErrorMessage ="Ýçerik alaný gereklidir")]
        public string Icerik { get; set; }

        public DateTime YayimTarihi { get; set; }

        public Nullable<Guid> YazarID { get; set; }

        public int KategoriID { get; set; }

        public int TipID { get; set; }
        [Column(TypeName = "image")]
        public byte[] Resim { get; set; }
        [Column(TypeName = "image")]
        public byte[] KucukResim { get; set; }        

        public int Goruntulenme { get; set; }

        public virtual aspnet_Users aspnet_Users { get; set; }

        public virtual Kategori Kategori { get; set; }

        public virtual Tip Tip { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<Resim> Resim1 { get; set; }
    }
}
